README.md

```diff
@@ -19,19 +19,24 @@ English | [简体中文](./i18n/README.zh-cn.md) | [繁體中文](./i18n/README.
 
 # QuestDB
 
-QuestDB is an open-source [time-series database](https://questdb.io/glossary/time-series-database/) for high throughput
+QuestDB is an open-source time-series database for high throughput
 ingestion and fast SQL queries with operational simplicity.
 
-QuestDB is well-suited for financial market data, IoT sensor data, application metrics, ad-tech, real-time dashboards, and fast analytics.
+QuestDB is well-suited for financial market data, IoT sensor data, ad-tech and real-time dashboards. It shines for datasets with [high cardinality](https://questdb.io/glossary/high-cardinality/)
+and is a drop in replacement of InfluxDB Line Protocol.
 
 QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL extensions make it simple to filter and downsample data, 
-or correlate data from multiple sources using relational and time-series joins. We achieve high performance by adopting a
-column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques. The entire
-codebase is built from the ground up in Java and C++, with no dependencies and zero garbage collection.
+or correlate data from multiple sources using relational and time-series joins. 
+
+We achieve high performance by adopting a column-oriented storage model, parallelized vector execution, SIMD instructions, and low-latency techniques. 
+The entire codebase is built from the ground up in Java, C++ and Rust (Enterprise) with no dependencies and zero garbage collection.
```

Comment:
This kind of makes it sound like the codebase is written in an Enterprise edition of Rust or something. Maybe we could use [footnote syntax](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/)?

Reply:
Yes true, its a bit confusing I agree. Footnote syntax works or alternatively we can remove (enterprise) and link https://questdb.io/blog/leveraging-rust-in-our-high-performance-java-database/ to Rust

Reply:
addressed here https://github.com/questdb/questdb/pull/4188/commits/103738de4f35c75ba0ac6d10a7a9a5c5d38c18e5